core/extension/gdextension_interface.cpp

```diff
@@ -1036,6 +1125,10 @@ static void gdextension_ref_set_object(GDExtensionRefPtr p_ref, GDExtensionObjec
 	ref->reference_ptr(o);
 }
 
+static GDExtensionObjectPtr gdextension_callable_custom_create(GDExtensionCallableCustomInfo *p_custom_callable_info) {
+	return (GDExtensionObjectPtr)memnew(CallableCustomExtension(p_custom_callable_info));
```

Comment:
Need to cast from `CallableCustomExtension` to `Callable`, possibility via placement new.

Reply:
Thanks! Fixed as described above in my latest push